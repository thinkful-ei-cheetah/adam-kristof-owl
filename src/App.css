.avatar-img, .avatar-image-large{
  border-radius: 50%;
  border: 1px solid grey;
}
.avatar-img{
  width:30px;
}
.avatar-image-large{
  width: 50px;
}