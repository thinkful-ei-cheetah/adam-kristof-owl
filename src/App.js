import React, { Component } from 'react';
import './App.css';
import Stage from './components/stage/Stage';
import ChatLog from './components/chat/ChatLog'
import ParticipantLog from './components/participants/ParticipantLog'

class App extends Component {
  state={
    showChat: false
  }

  render() {

    console.log(this.props)

    return (
      <div className="App">
        <div className="sidebar">
          <ChatLog />
        </div>
        <div className="video-box">
          <Stage/>
        </div>
        {/* Bottom Bar can be another component made up of more components or flexbox? */}
        
      </div>
    );
  }
}

export default App;
