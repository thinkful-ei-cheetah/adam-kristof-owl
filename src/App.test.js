import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App participants={[]} chatData={[]}/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Renders as it did the first time!', ()=>{
  const tree = renderer
  .create(<App participants={[]} chatData={[]}/>)
  .toJSON()
  expect(tree).toMatchSnapshot()  
;
});