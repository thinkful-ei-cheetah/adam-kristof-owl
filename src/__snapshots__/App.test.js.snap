// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders as it did the first time! 1`] = `
<div
  className="App"
>
  <div
    className="sidebar"
  >
    <div
      className="participentView"
    >
      <ul />
    </div>
    <div
      className="ChatLog"
    >
      <ul />
    </div>
  </div>
  <div
    className="video-box"
  >
    <div
      className="vid"
    >
      <ul />
    </div>
  </div>
</div>
`;
